index.html

```diff
@@ -0,0 +1,167 @@
+                <h2>ATTACK ON TITAN<br>THE FINAL SEASON<br>PART 2</h2>
+                <p class="info-front">
+                    The Final Season (Part 2), which is scheduled to air in the fall of 2023!
+                    In the new PV, Armin recalls the memories of the three of them when they were young, and the line
+                    "Goodbye Ellen" of Mikasa who was also in the key image is symbolic!
+                </p>
+                <h3>Release date: October 2023</h3>
+                <p>Platform release: crunchyroll - wit studio NHK</p>
+            </div>
+        </section>
+    </header>
+    <main>
+        <section id="section1" class="section1">
+            <div class="cont-ti">
+                <h2>Main Program</h2>
+            </div>
+            <hr class="line">
+            <ul class="card-container">
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr1.png" alt="">
+                        <h3>Exhibition</h3>
+                        <p>Discover the "Latest Releases" in movies and series, find the most popular productions and receive recommendations.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr2.png" alt="">
+                        <h3>Forum</h3>
+                        <p>Participa en "Discusiones generales", analiza episodios, especula sobre tramas, lee reseñas de anime y comparte fanart y cosplay.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr3.png" alt="">
+                        <h3>Network</h3>
+                        <p> Create your profile, send private messages, join groups, check the events calendar and find friends with similar interests.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr4.png" alt="">
+                        <h3>Lecture</h3>
+                        <p>Browse the schedule, meet guest speakers, participate in Q&A sessions, and enjoy live streams.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr5.png" alt="">
+                        <h3>Workshop</h3>
+                        <p>Learn with featured instructors, sign up for specific times, and access additional resources and materials.</p>
+                    </div>
+                </li>
+            </ul>
+            <div class="cont-des">
+                <h2 class="mo"><a href="#">SEE MORE</a></h2>
+                <button type="button" class="des">Join into the world of anime</button>
+            </div>
+        </section>
```

Comment:
- [ ] As stated in the project requirement The pages should look almost identical to the original design, outside the content differences. So Kindly I recommend you to  **adjust the programs in one rows and remove the space between the cards** just like the Figma design But in the case of your the programs list are in two rows and there is gap between the cards, So please adjust it.
- The Figma Main program section Look likes this : 
![main program](https://github.com/AndresEspi/animePage/assets/93598189/ab63f5bd-eb92-41d6-8b14-4d3c24f9ea69)

- Your Main program design Look likes this : 
![main program](https://github.com/AndresEspi/animePage/assets/93598189/7dd925f9-5c66-4b6d-8d8b-a360e64a5516)